awx/ui/src/components/HealthCheckButton/HealthCheckButton.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { Plural, t } from '@lingui/macro';
+import { Button, DropdownItem, Tooltip } from '@patternfly/react-core';
+import { useKebabifiedMenu } from 'contexts/Kebabified';
+
+function HealthCheckButton({ isDisabled, onClick, selectedItems }) {
+  const { isKebabified } = useKebabifiedMenu();
+  const hopNodeSelected =
+    selectedItems.filter((instance) => instance.node_type === 'hop').length > 0;
+  const hasSelectedItems = selectedItems.length > 0;
+
+  const buildTooltip = () => {
+    if (hopNodeSelected) {
+      return (
+        <Plural
+          value={hopNodeSelected}
```

Comment:
```suggestion
          value={hopNodeSelected}
```

hopNodeSelected is a boolean,

Reply:
Good catch.  This should be selectedItems.length